.github/workflows/dataflow.yaml

```diff
@@ -0,0 +1,48 @@
+name: Dataflow Integration Test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    types: [ opened, reopened, synchronize, labeled ]
+
+jobs:
+  build:
+    name: ${{ matrix.python-version }}
+    # run on:
+    #  - all pushes to main
+    #  - a PR was just labeled 'test-dataflow'
+    #  - a PR with 'test-dataflow' label was opened, reopened, or synchronized
+    if: |
+      github.event_name == 'push' ||
+      github.event.label.name == 'test-dataflow' ||
+      contains( github.event.pull_request.labels.*.name, 'test-dataflow')
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9"]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
```

Comment:
Can we make this something more specific, like `GCP_DATAFLOW_SERVICE_KEY`?

Reply:
Good call. Fixed in 5523840.